change/@ni-nimble-components-df1cc26c-9976-477a-8fa0-f9b72492eaa0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
One behavior I find a bit unintuitive is that using up and down I can see full row selection for rows and group header rows. Using right I can enter a row and navigate cells and using left I can go all the way left and then exit cell nav back to row nav. But on group header rows pushing right and then left does not let me re-enter row nav.
![re-enter row on left arrow](https://github.com/ni/nimble/assets/1588923/119a8e32-6ce5-44dc-86d0-ac7168e3dfb2)
![re-enter row on left arrow](https://github.com/ni/nimble/assets/1588923/119a8e32-6ce5-44dc-86d0-ac7168e3dfb2)

Reply:
Investigated and this was a bug with group row keyboard nav behavior. 

Now fixed (LeftArrow from the group row selection checkbox should refocus the whole row).